package/tests/test_server.py

```diff
@@ -102,6 +103,14 @@
 }
 
 
+def get_pipeline(name: str = None):
```

Comment:
```python
def get_pipeline(name: str = "__default__"):
    if name == "__default__":
        return create_pipeline()
    if name == "another":
        return Pipeline([])
    raise KeyError("Failed to find the pipeline.")
```

Reply:
I meant to mock the original `get_pipeline` implementation (not exactly the same though), and I think the current one looks closer to the original one :)